ui/main.qml

```diff
@@ -208,19 +207,6 @@ StatusWindow {
 
         loader.sourceComponent = app
 
-        if(localAccountSensitiveSettings.recentEmojis === "") {
-            localAccountSensitiveSettings.recentEmojis = [];
-        }
-        if (localAccountSensitiveSettings.hiddenCommunityWelcomeBanners === "") {
-            localAccountSensitiveSettings.hiddenCommunityWelcomeBanners = [];
-        }
-        if (localAccountSensitiveSettings.hiddenCommunityBackUpBanners === "") {
-            localAccountSensitiveSettings.hiddenCommunityBackUpBanners = [];
-        }
-
-        Theme.changeTheme(localAppSettings.theme)
-        Theme.changeFontSize(localAccountSensitiveSettings.fontSize)
```

Comment:
Moved to AppMain, this was only executed after loading it

Reply:
The theme was executed on app start no?